serialization/src/TorchForceProxy.cpp

```diff
@@ -55,9 +96,24 @@ void TorchForceProxy::serialize(const void* object, SerializationNode& node) con
 }
 
 void* TorchForceProxy::deserialize(const SerializationNode& node) const {
-    if (node.getIntProperty("version") != 1)
+    int storedVersion = node.getIntProperty("version");
+    if (storedVersion > 2)
         throw OpenMMException("Unsupported version number");
-    TorchForce* force = new TorchForce(node.getStringProperty("file"));
+    TorchForce* force;
+    if (storedVersion == 1) {
+        string fileName = node.getStringProperty("file");
+        force = new TorchForce(fileName);
+    } else if (storedVersion == 2) {
```

Comment:
I would just make this `else`, not `else if`.  This is presumably how all future versions will load it as well.